assets/app/views/directives/labels.html

```diff
@@ -1,33 +1,11 @@
-<osc-form-section
-  header="Labels"
-  about-title="labels"
-  about="Labels are used to organize, group, or select objects and resources, such as pods."
-  expand="expand"
-  can-toggle="canToggle"
-  >
-  <div ng-if="(labels | hashSize) !== 0 || $parent.expand" class="help-block">Each label is applied to each created resource.</div>
-  <div ng-show="expand">
-
-    <osc-key-values
-      key-title="Name"
-      entries="labels"
-      key-validator="label"
-      value-validator="label"
-      delete-policy="added"
-      key-validation-tooltip="A valid object label has the form [domain/]name where a name is an alphanumeric (a-z, and 0-9) string, with a maximum length of 63 characters, with the '-' character allowed anywhere except the first or last character. A domain is a sequence of names separated by the '.' character with a maximum length of 253 characters."
-      value-validation-tooltip="A valid label value is an alphanumeric (a-z, and 0-9) string, with a maximum length of 63 characters, with the '-' character allowed anywhere except the first or last character."
-      readonly-keys="template"
-      >
-
-    </osc-key-values>
-  </div>
-  <div ng-hide="expand">
-    <osc-key-values
-      key-title="Labels"
-      entries="labels"
-      editable="false"
-      can-toggle="canToggle"
-      >
-    </osc-key-values>
-  </div>
-</osc-form-section>
+<div ng-if="(labels | hashSize) > 0">
+  <span ng-repeat="(labelKey, labelValue) in labels" class="k8s-label" ng-if="!limit || $index < limit">
```

Comment:
limitToOrAll didn't seem to work on hashes, i think limitTo only works on
arrays

On Thu, Oct 15, 2015 at 11:18 AM, Sam Padgett <notifications@github.com>
wrote:

> In assets/app/views/directives/labels.html
> <https://github.com/openshift/origin/pull/4976#discussion_r42137551>:
>
> > -      >
> > -
> > -    </osc-key-values>
> > -  </div>
> > -  <div ng-hide="expand">
> > -    <osc-key-values
> > -      key-title="Labels"
> > -      entries="labels"
> > -      editable="false"
> > -      can-toggle="canToggle"
> > -      >
> > -    </osc-key-values>
> > -  </div>
> > -</osc-form-section>
> > +<div ng-if="(labels | hashSize) > 0">
> > +  <span ng-repeat="(labelKey, labelValue) in labels" class="k8s-label" ng-if="!limit || $index < limit">
>
> (labelKey, labelValue) in labels | limitToOrAll : limit instead of the
> ng-if?
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/openshift/origin/pull/4976/files#r42137551>.
>